chrome-extension/test/configure.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+process.exit();
```

Comment:
It did not, but I could fix this in a PR later (if absolutely necessary). That commit was followed by another one to disable the chromium test. The scope of this PR is to fix #87, and enabling the test is and addition.